nixos/modules/services/continuous-integration/github-runner/service.nix

```
@@ -153,18 +157,21 @@ with lib;
             ln -s '${newConfigPath}' "${currentConfigPath}"
           fi
         '';
-        setupRuntimeDir = writeScript "setup-runtime-dirs" ''
+        setupWorkDir = writeScript "setup-work-dirs" ''
+          # Cleanup previous service
+          rm -rf "$WORK_DIRECTORY/*"
```

As far as I can tell, this won't delete hidden dot files. Apart from that, the shell won't glob the `*` inside the quotes.

```suggestion
          ${pkgs.findutils}/bin/find "$WORK_DIRECTORY" -mindepth 1 -delete
```

---

Done, thanks!  Forgot about this!